TeamMandate/Mandate.md

```diff
@@ -12,17 +12,13 @@ This will be done through an open collaborative approach that will further ESDC'
 
 ## Mandate (team's authority within IITB)
 
-- Manage IT Strategic Evolution Fund 
-- Full ownership and governance of projects operating against the IT Strategic Evolution fund
-- Provide recommendations to senior management on how IT can be leveraged to improve ESDC's mandate delivery 
-- Conduct both inward facing and external research in order to provide evidence based aforementioned recommendations 
-- Permeate and enforce modern IT practices and processes 
-- Provide strategic direction to IITB's initiatives in order to maximize their impacts and transition IITB into a holistic service-oriented organization
-- Produce IT strategies that includes diagnosis, guiding policies, and sets of coherent actions, for IITB to continually improve its responsiveness to business change
-- Channel senior level support to reprioritize tasks to align ESDC to the approved vision
-- Continuously monitor technology and policy trends to keep pace with technological development (such as OECD AI Policy Observatory)
-- Forge partnerships with key IM/IT partners and suppliers, including across jurisdictions and private and public institutions, for utilizing their expertise and knowledge to meet service delivery goals
-- Orchestrate the IT ecosystem to bring strategies to completion
+- Authority on Open Source documentation 
+- Report to senior management on digital strategic objectives of ESDC
+- Report to senior management on compliance with TBS directives 
+- Authority to merge working groups, or committees, within IITB
+- Report to senior management on our adherence to technological and policy trends (such as OECD AI Policy Observatory)
+- Conduct inwards facing analysis of ESDC's IT processes
```

Comment:
Analysis? Can we have a carrot and stick to actually change those processes or are the owners still in charge of making those changes?

Reply:
This point is actually defined in the Briefing action above.